File: CallByReference/CallByReference8.java
public class CallByReference8 {

    public void swap(int x, int y) {
        int t = x;
        x = y;
        y = t;
        System.out.println("Inside swap(): " + x + " " + y);
    }

    public static void main(String[] args) {
        CallByReference8 obj = new CallByReference8();
        int x = 10, y = 20;

        System.out.println("Before swap(): " + x + " " + y);
        obj.swap(x, y);
        System.out.println("After swap(): " + x + " " + y);
    }
}

File: CallByReference/CallByReference.8.js
function swap(x, y) {
    var t = x;
    x = y;
    y = t;
}

let x = 10, y = 20;
console.log('Before swap():' + x + ' ' + y)
swap(x, y)
console.log('After swap():' + x + ' ' + y)
